src/views/catalog/wizard/tabs/disks/WizardDisksTab.tsx

```diff
@@ -41,8 +40,9 @@ const WizardDisksTab: WizardTab = ({ loaded, tabsData, updateTabsData, updateVM,
           pathsToHighlight={PATHS_TO_HIGHLIGHT.DISKS_TAB}
           resource={vm}
         >
-          <DiskListTitle />
-
+          <Flex className="wizard-disk-tab__flex">
```

Comment:
u removed DiskListTitle component, can this file be removed completely? any other component is using it?

Reply:
> u removed DiskListTitle component, can this file be removed completely? 

No.

> any other component is using it?

Yes. VMs and Templates Disks tabs.

Reply:
shouldn't we be consistent in those 2 places also? why is only wizard a different?

Reply:
Check the jira ticket, please. Thanks.

Reply:
yes i did check this, it's a jira ticket you opened so you should know best. 
Because of this PR we have 2 separate looks one for wizard and second for templates and vms. why is that? In the jura it only referred to a clean wizard. Why no info about maybe fixing templates and vms to be consistent? you had 2 solutions there - make as vms and template or change to a clean look. They go for a clean look, but no referring if it should be applied also for vms and templates - this is exactly why I'm asking here - if this look should be applied to others as well? (vms and templates)

Reply:
@metalice Hi :) 
Yifat and I were also discussing this topic.
the reason for different looks is because under the customize wizard the title is not really needed as each page has a single context which is the tab's text.
However some template and vm tabs have more than one thing in there page (disks tab has disks table and filesystem table for example) and Yifat felt the titles were needed.
I hope that makes sense to you :)